### ISSUE TITLE
xdstool 6.0.2 - cannot download conformance test log

### ISSUE THREAD

**Issue by kevingill1966** [usnistgov/iheos-toolkit2#352]
I am running pre-connectathon tests for Hague Connectathon. As part of this I have a number of tests to run in the xdstools tool for XCA Initiating Gateway. I am running a local copy of xdstools to do the tests. (I think this is the required approach).

I downloaded a docker builder for xdstools - 	https://github.com/ahdis/xdstools-docker.

I downloaded the 6.0.2 war file (I tried 6.1.0 but couldn't get default enviroment working).

I logged in. I configured the cache to write to a directory. I added a test session (named session1).

I sent through the first test message (test: 12300).

I clicked on the link to view the session1 community1 simulator log.

I found the correct message. I am supposed to mark the 

I tried to download the message. This is the url: http://localhost:8080/xdstools6/message/session1__community1/reg/sq/2018_04_08_18_11_39_202

This shows a Java 500 error on the screen, and this in the tomcat logs.

```
Initializing toolkit in SimServlet
...warHome is /usr/local/tomcat/webapps/xdstools6
... oops - warHome already initialized to /usr/local/tomcat/webapps/xdstools6
...warHome initialized to /usr/local/tomcat/webapps/xdstools6
ServletContext initialized to /xdstools6
ServletContext initialized to /xdstools6
Initializing AdtServlet
Initializing ADT Listeners...
Port range is [5000, 5015]
AdtServlet initialized
Loading runtime for sim default__community2 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for default__community2 on port 5001
Launching listener for simId default__community2 on port 5001
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
Loading runtime for sim default__community1 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for default__community1 on port 5000
Launching listener for simId default__community1 on port 5000
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
Loading runtime for sim session1__community2 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for session1__community2 on port 5003
Launching listener for simId session1__community2 on port 5003
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
Loading runtime for sim session1__community1 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for session1__community1 on port 5002
Launching listener for simId session1__community1 on port 5002
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
Loading runtime for sim default__community2 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for default__community2 on port 5001
Launching listener for simId default__community2 on port 5001
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
Loading runtime for sim default__community1 of type rg of class gov.nist.toolkit.fhir.simulators.sim.rg.RGADActorSimulator
Create V2 PIF listener for default__community1 on port 5000
Launching listener for simId default__community1 on port 5000
Available ports are [5004, 5005, 5006, 5007, 5008, 5009, 5010, 5011, 5012, 5013, 5014, 5015]
SimServlet initialized
Proxy Operation: start proxy from SimServlet on thread 21 port 7297
Proxy Operation: Starting Sim Proxy on port 7297
Starting proxy listener thread on 7297
Proxy startup on port 21 failed - Address already in use (Bind failed)
Exception BindException
	at gov.nist.toolkit.fhir.simulators.proxy.service.RequestListenerThread.<init>(RequestListenerThread.java:46)
	at gov.nist.toolkit.fhir.simulators.proxy.service.ElementalReverseProxy.start(ElementalReverseProxy.java:63)
	at gov.nist.toolkit.fhir.simulators.proxy.service.ElementalReverseProxy.start(ElementalReverseProxy.java:70)
	at gov.nist.toolkit.fhir.simulators.servlet.SimServlet.init(SimServlet.java:100)

```

**Comment by iheos** [comment]
This is fixed in 6.1.1 which will be out on Monday (9 April)

**Comment by iheos** [comment]
Fixed
